test/integration/targets/win_module_utils/library/command_util_test.ps1

```diff
@@ -92,13 +92,13 @@ Assert-Equals -actual $actual.stdout -expected "stdout `r`n"
 Assert-Equals -actual $actual.stderr -expected "stderr `r`n"
 
 $test_name = "Test UTF8 output from stdout stream"
-$actual = Run-Command -command "powershell.exe -ExecutionPolicy ByPass -Command `"Write-Host '💩'`""
+$actual = Run-Command -command "powershell.exe -ExecutionPolicy ByPass -Command `"Write-Output '💩'`""
```

Comment:
This is mostly unneeded, while `Write-Host` is usually the wrong thing to do, in this case we are executing a new process and capturing the stdout ourselves which will include the host output values. We want to avoid the PowerShell output pipeline for this test.